### PR TITLE
Multi topics

### PR DESCRIPTION
**Pull request by eno-intel**
multi topic support and config for unwrapped messages

### CODE REVIEW

File: cmd/res/configuration.toml [hunk 1]
```diff
@@ -44,6 +44,22 @@ LogLevel = "INFO"
...
+# Pre-define Devices
+[[DeviceList]]
+  Name = "IntelAcePoint"
+  Profile = "MQTT-Device"
+  Description = "MQTT device"
+  Labels = [ "MQTT"]
+  [DeviceList.Protocols]
+    [DeviceList.Protocols.mqtt]
+       Schema = "tcp"
+       Host = "RRS-Rapid-ElasticL-Q3I1D93JN2JQ-325700436.us-west-2.elb.amazonaws.com"
```
Comment (adam-intel) [inline cmd/res/configuration.toml]: This host should probably be changed to localhost

File: cmd/res/configuration.toml [hunk 2]
```diff
@@ -52,25 +68,25 @@ LogLevel = "INFO"
...
   RemoveCmd = ""
   RemoveCmdArgs = ""
-  ProfilesDir = ""
+  ProfilesDir = "./res"
 
 # Driver configs
 [Driver]
 IncomingSchema = "tcp"
-IncomingHost = "0.0.0.0"
+IncomingHost = "RRS-Rapid-ElasticL-Q3I1D93JN2JQ-325700436.us-west-2.elb.amazonaws.com"
```
Comment (adam-intel) [inline cmd/res/configuration.toml]: localhost here as well.

File: internal/driver/driver.go [hunk 3]
```diff
@@ -26,22 +26,22 @@ import (
...
-	MqttBroker     string
-	MqttBrokerPort int
-	MqttClientID   string
-	MqttTopic      string
-	MqttQos        int
-	MqttUser       string
-	MqttPassword   string
-	MqttKeepAlive  int
-}
+// type Config struct {
```
Comment (adam-intel) [inline internal/driver/driver.go]: Should be removed

**Review by adam-intel — COMMENTED** [review]
Overall is good but I think we should some unit tests for the new config stuff we added.

**Review by lenny-goodell — COMMENTED** [review]
Agree with Adam's comments. Nothing to add.

**Comment by eno-intel** [comment]
will work on unit tests

**Pull request merged**